components/camera/videosource/helpers.go

```diff
@@ -22,6 +23,11 @@ import (
 // detected by http.DetectContentType.
 func getMIMETypeFromData(ctx context.Context, data []byte) (string, error) {
 	detectedMimeType := http.DetectContentType(data)
+	if !strings.Contains(detectedMimeType, "image") {
+		// TODO: image.RegisterFormat "text/plain" to parse error messages
```

Comment:
Can create a ticket for this or this in an `init` function. Plain text can be decoded by simply casting the bytes to a string. That is
```
var myStr string
myStr = string(data) // the data is now a string
```

Reply:
When this is an error response, it would be nice to include the error text in the returned error here.

Reply:
Done!